docs/react-tutorial.md

```diff
@@ -0,0 +1,340 @@
+<img src="https://www.transposit.com/img/transposit-logo-black.png" width="182px" alt="Transposit"/>
+
+# React Tutorial
+
+Build a web app using [React](https://reactjs.org/) and [Netlify](https://www.netlify.com/) as the frontend, and [Transposit](https://www.transposit.com/) as the backend.
+
+> If you're new to Transposit, browse [its quickstart](https://www.transposit.com/docs/get-started/quickstart/). This React tutorial will assume basic familiarity with Transposit.
+
+# Set up Transposit
+
+[Sign in to Transposit](https://console.transposit.com).
+
+## Create a new app
+
+Create a new app and give it a name, such as `hello_react`.
```

Comment:
From @pmoayer : You may want "Create a new app..." to link to console.transposit.com/apps/new since it's still the Slack template default